persistence-model/src/main/resources/application-dev.conf

```diff
@@ -1,18 +1,37 @@
 akka {
   loglevel = "INFO"
-  actor.warn-about-java-serializer-usage = off
+  actor {
+    warn-about-java-serializer-usage = off
+    serializers {
+      institution = "hmda.persistence.serialization.institutions.InstitutionProtobufSerializer"
+    }
+    serialization-bindings {
+      "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionCreated" = institution
+      "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionModified" = institution
```

Comment:
Yes, that is right. These two events are what get persisted as part of `InstitutionPersistence`.